src/api/app/presenters/notification_presenter.rb

```diff
@@ -37,7 +45,7 @@ def excerpt
             else
               ''
             end
-    text.to_s.truncate(100)
+    truncate_to_first_new_line(text.to_s)
```

Comment:
`truncate_to_first_new_line` could be replaced with:
```suggestion
    truncation_ellipsis_length = 3
    truncation_index = (text.index("\n").try(:+, truncation_ellipsis_length) || TRUNCATION_LENGTH).clamp(0, TRUNCATION_LENGTH)
    text.truncate(truncation_index)
```

Reply:
Thanks for the suggestion, but:
- I find the alternative not so easy to read, and
- I have done some profiling, and your code seems to be between 30% and 40% slower.

I can show you what I have profiled offline.